lib/taskbook.js

```diff
@@ -325,8 +325,50 @@ class Taskbook {
     render.successCreate(task);
   }
 
+  _flattenDown(array, result) {
+    for (let i = 0; i < array.length; i++) {
+      const value = array[i];
+
+      if (Array.isArray(value)) {
+        this._flattenDown(value, result);
+      } else {
+        result.push(value);
+      }
+    }
+
+    return result;
+  }
+
+  _generateNumbers(value) {
+    let _start = value[0] || null;
+    let _end = value[1] || null;
```

Comment:
Should this still work if the user inputs  `taskbook -d 1-5-10`?
Looks like it will currently just include 1-5, but maybe an error should be returned, or 1-10 should be used instead?

Reply:
You should use ` 1-10 ` instead of `taskbook -d 1-5-10`